src/bindings/python/BUILDING.md

```diff
@@ -1,248 +1,35 @@
-
-    %OPENVINO_BASEDIR%\openvino_dist\setupvars.bat
-
-### Build a Python Wheel on Windows
-
-Build the Python wheel package:
-
-    cd "%OPENVINO_BASEDIR%/openvino/src/bindings/python"
-    python setup.py bdist_wheel
-
-The final wheel should be located in the `bindings\python\dist` directory.
-
-    dir openvino/src/bindings/python/dist/
-    10/09/2020  04:06 PM         4,010,943 openvino-0.0.0.dev0-cp<version>-cp<version>-win_amd64.whl
-
-## Run Tests
-
-### Use a virtualenv (Optional)
+- ### Linux* Systems
+    Follow instructions available on [GitHub wiki pages for Linux build](https://github.com/openvinotoolkit/openvino/wiki/BuildingForLinux).
 
-You may wish to use a virutualenv for your installation.
+    Exact Python instructions can be found in [Additional Build Options section](https://github.com/openvinotoolkit/openvino/wiki/BuildingForLinux#additional-build-options).
 
-    $ virtualenv -p $(which python3) venv
-    $ source venv/bin/activate
-    (venv) $
+- ### Windows* Systems
+    Follow instructions available on [GitHub wiki pages for Windows build](https://github.com/openvinotoolkit/openvino/wiki/BuildingForWindows).
 
-### Install the Wheel and Other Requirements
+    Exact Python instructions can be found in [Additional Build Options section](https://github.com/openvinotoolkit/openvino/wiki/BuildingForWindows#additional-build-options).
 
-    (venv) $ cd "${OPENVINO_BASEDIR}/openvino/src/bindings/python"
-    (venv) $ pip3 install -r requirements.txt
-    (venv) $ pip3 install -r requirements_test.txt
-    (venv) $ pip3 install dist/openvino-0.0.0.dev0-cp38-cp38-linux_x86_64.whl
+- ### macOS* Systems for Intel CPU
+    Follow instructions available on [GitHub wiki pages for macOS build](https://github.com/openvinotoolkit/openvino/wiki/BuildingForMacOS_x86_64).
 
-### Run Tests
+- ### macOS* Systems for Apple Silicon
+    Follow instructions available on [GitHub wiki pages for macOS build](https://github.com/openvinotoolkit/openvino/wiki/BuildingForMacOS_arm64).
 
-You should now be able to run tests.
+- ### Raspbian Stretch* OS
+    Follow instructions available on [GitHub wiki pages for Raspbian Stretch OS build](https://github.com/openvinotoolkit/openvino/wiki/BuildingForRaspbianStretchOS).
 
-You may need to run the `setupvars` script from the OpenVINO™ Toolkit to set paths to OpenVINO™ components.
+    Exact Python instructions can be found in [Additional Build Options section](https://github.com/openvinotoolkit/openvino/wiki/BuildingForRaspbianStretchOS#additional-build-options).
 
-    source ${OPENVINO_BASEDIR}/openvino_dist/setupvars.sh
+## Run tests to verify OpenVINO™ Python API
 
-Now you can run tests using `pytest`:
+Follow instructions in [How to test OpenVINO™ Python API?](./docs/test_examples.md#Running_OpenVINO™_Python_API_tests) to verify the build.
 
-    pytest tests
+*Note: You may need to run the `setupvars` script from the OpenVINO™ Toolkit to set paths to OpenVINO™ components.*
```

Comment:
Actually it makes sense to run `setupvars` script if only `install` command was executed. May be add something about it?